tests/acceptance/generate.component.spec.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
```

Comment:
A minor thing, but shouldn't the file be named `generate-component.spec.js`? We overall go for dash separated file names, and reserve the dots to separate functionality groups.